### PR TITLE
Add html & css file

### PR DESCRIPTION
**Pull request by fickryiman**
create index.html
create styles.css file
add html-css linters.yml configuration

### CODE REVIEW

**Review by Zuhaib042 — CHANGES_REQUESTED** [review]
Hi @fickryiman , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
required changes

## Highlights
✅ Correct Github flow

## Required Changes ♻️
_Check the comments under the review._
- [x] Good job setting up the linters in your project but the problem is that they are not passing. First of all, you don't need the `ESlint` in html-css projects. kindly add just [these linters](https://github.com/microverseinc/linters-config/tree/master/html-css) in your project. 
second, after installing linters in your project you can check them by running `npx hint .` and `npx stylelint "**/*.{css,scss}"` and see if they are passing or not. 
- [x] Our project needs to pass all linters in order to get approved. so kindly consider passing them for approval of this pull request.
- [x] To make our project looks professional, we need to add a descriptive title for the pull request and a summary under title with meaningful points describing what changes are inside our pull request. kindly consider applying this change in order to proceed.
- [x] The README file has following issues: 
  - [x] you can add HTML and CSS as client instead of `react`. and you can remove `server` and `database` as there are none for this project.
  - [x] Kindly consider adding proper meaningful features in features section like `adding styles to html`
  - [x] Kindly consider removing the `live demo` link as there is none.
  - [x] Kindly consider removing the second `author` as there is none. 
## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- x
- y
- z

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by fickryiman** [comment]
Please re-review has fixes and do all the advices.

File: README.md [hunk 1]
```diff
@@ -37,21 +37,22 @@
...
+    <li><a href="#"></a></li>
   </ul>
 </details>
 
 <details>
 <summary>Database</summary>
   <ul>
-    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+    <li><a href="#"></a></li>
   </ul>
```
Comment (Shedrack-Sunday) [inline README.md]: - [x] Just as the previous reviewer said, instead of leaving the server and database pace blank, please kindly remove it 👍

**Review by Shedrack-Sunday — CHANGES_REQUESTED** [review]
Hi @fickryiman ,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
required changes

Highlights
✅ Correct Github flow

Required Changes ♻️
Check the comments under the review.

- [x] You seem to still be missing some linter files.  kindly add just [these linters](https://github.com/microverseinc/linters-config/tree/master/html-css) in your project.

## Linters needed:

image

 - [x] As suggested by the previous reviewer, please add a  descriptive title for the pull request and a summary under the title with meaningful points describing what changes are inside our pull request. 
 
 ## Pull request header :
 
image

 
 The README file has the following issues:

 - [x] Kindly add  proper meaningful features in the features section like adding styles to HTML
 
Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

You can also consider:

x
y
z
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: README.md (hunk 1 above)
Comment (fickryiman) [reply README.md]: Done, fixes all the issues. Please kindly help to review and approval.

**Comment by fickryiman** [comment]
Fixes all the issues, Please review for last chance, i hope this time i have an approval. Thank you reviewer, you teach me with another ways.

**Comment by Shedrack-Sunday** [comment]
Your branch still has conflicts. kindly resolve it and send your PR on your dashboard 👍

File: .gitignore [hunk 2]
```diff
@@ -1 +1,2 @@
-node_modules
\ No newline at end of file
+node_modules/
+test.md
```
Comment (wayungi) [inline .gitignore:2]: - [ ] Good work adding file s to the .ignore 👏🏾

File: README.md [hunk 3]
```diff
@@ -60,9 +47,9 @@
 ### Key Features <a name="key-features"></a>
 
 <!-- > Describe between 1-3 key features of the application. -->
-- **[key_feature_1]**
-- **[key_feature_2]**
-- **[key_feature_3]**
+- **[adding index.html file]**
+- **[adding styles to html]**
+- **[adding submit button form]**
```
Comment (wayungi) [inline README.md]: - [OPTIONAL ] You can take the square brackets away when you are done editing this section. 🤪

**Review by wayungi — APPROVED** [review]
Hi @fickryiman , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
image

## Highlights
- [x] Professional README ✔️ 
- [x] All linter checks passing ✔️ 
- [x] Good commit messages ✔️ 
- [x]  Correct GitHub flow
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 3 above)
Comment (wayungi) [reply README.md]: - [OPTIONAL ] Please resolve this conflict so that all your files are up to date

File: .gitignore (hunk 2 above)
Comment (fickryiman) [reply .gitignore:2]: Thank you

File: README.md (hunk 3 above)
Comment (fickryiman) [reply README.md]: Noted, i will fix this by remove the square brackets

**Comment by fickryiman** [comment]
merged successfully, previously resolve conflicts on readme file.

**Pull request merged**